streams/src/test/java/org/apache/kafka/streams/processor/internals/StandbyTaskTest.java

```diff
@@ -115,6 +119,7 @@ private StreamsConfig createConfig(final File baseDir) throws Exception {
                 setProperty(StreamsConfig.BUFFERED_RECORDS_PER_PARTITION_CONFIG, "3");
                 setProperty(StreamsConfig.STATE_DIR_CONFIG, baseDir.getCanonicalPath());
                 setProperty(StreamsConfig.TIMESTAMP_EXTRACTOR_CLASS_CONFIG, MockTimestampExtractor.class.getName());
+                setProperty(StreamsConfig.COMMIT_INTERVAL_MS_CONFIG, "1");
```

Comment:
Since we are using MockTime, we do not need to enforce the internal to be very small but still can run it as fast as we want, right?